From 8022a71009afce2a1dcac9c68491f846de12e54b Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Tue, 15 Jun 2021 10:20:15 +0800
Subject: [PATCH] drivers: spi: fsl-dspi: delete the unused variable

This patch is to fix the following build warning:

	kernel-source/drivers/spi/spi-fsl-dspi.c:937:15:warning: unused variable 'val' [-Wunused-variable]
	937 | unsigned int val;

The use of variable val has been dropped, so delete it to fix this warning.

Upstream-Status: pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/spi/spi-fsl-dspi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/spi/spi-fsl-dspi.c b/drivers/spi/spi-fsl-dspi.c
index a81960e30ada..99724e2ceeb3 100644
--- a/drivers/spi/spi-fsl-dspi.c
+++ b/drivers/spi/spi-fsl-dspi.c
@@ -934,7 +934,6 @@ static int dspi_transfer_one_message(struct spi_controller *ctlr,
 	struct spi_device *spi = message->spi;
 	struct spi_transfer *transfer;
 	int status = 0;
-	unsigned int val;
 
 	message->actual_length = 0;
 
-- 
2.25.1

